mythril/interfaces/cli.py

```diff
@@ -364,10 +365,16 @@ def main():
             print(storage)
 
         elif args.disassemble:
-            easm_text = mythril.contracts[
-                0
-            ].get_easm()  # or mythril.disassemble(mythril.contracts[0])
-            sys.stdout.write(easm_text)
+            output = ""
+            easm_text = mythril.contracts[0].get_easm()
+            # or mythril.disassemble(mythril.contracts[0])
+
+            if easm_text:
```

Comment:
```suggestion
            if mythril.contracts[0].code::
```

Reply:
And then you can move `easm_text = mythril.contracts[0].get_easm()` to this if statement